src/app/app.component.html

```diff
@@ -1 +1,70 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<mat-toolbar role="navigation">
+  <div class="logo">
+    <mat-icon>fact_check</mat-icon>
+    <h1>TaskManager</h1>
+  </div>
+  <div class="nav-links">
+    <button class="desktop-button" mat-button routerLink="/tasks/create-task">
+      <mat-icon>add_task</mat-icon>
+      CREATE TASK
+    </button>
+    <button class="desktop-button" mat-button routerLink="/tasks/task-list">
+      <mat-icon>list</mat-icon>
+      TASK LIST
+    </button>
+    @if (this.authService.currentUser()?.roles?.includes("Admin")) {
```

Comment:
you can extract this into a variable so you don't have to write it over and over again instead you can use a simple variable like isAdmin or hasAdminRole